servlet/application.properties

```diff
@@ -202,6 +202,8 @@ suit.attivio.users=
 # the URL might be "https://.myhost.com/searchui".
 #saml.sso.metadata-generator.entity-base-url=
 
+# The page to navigate to if an unhandled authentication error
+#saml.sso.defaultFailureUrl=/autherror.html
```

Comment:
I don't see any reason to not have this uncommented by default... (e.g., active).